src/contracts/interfaces/IStakeRegistry.sol

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "./IRegistry.sol";
+
+/**
+ * @title Interface for a `Registry` that keeps track of stakes of operators for up to 256 quroums.
+ * @author Layr Labs, Inc.
+ */
+interface IStakeRegistry is IRegistry {
+    // DATA STRUCTURES
+
+    /// @notice struct used to store the stakes of an individual operator or the sum of all operators' stakes, for storage
+    struct OperatorStakeUpdate {
+        // the block number at which the stake amounts were updated and stored
+        uint32 updateBlockNumber;
+        // the block number at which the *next update* occurred.
+        /// @notice This entry has the value **0** until another update takes place.
+        uint32 nextUpdateBlockNumber;
+        // stake weight for the quorum
+        uint96 stake;
+    }
+
+    /**
+     * @notice Registers the `operator` with `operatorId` for the specified `quorumNumbers`.
+     * @param operator The address of the operator to register.
+     * @param operatorId The id of the operator to register.
+     * @param quorumNumbers The quorum numbers the operator is registering for.
+     * @dev Permissioned by RegistryCoordinator
```

Comment:
IMO this is kind of confusing wording. I would suggest 'permissioned to' or 'access restricted to the'

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/48/commits/7a220382b4fdbe6880cbec7103c3e5db6a036f24